lib_static/plugins/acts_as_customizable/lib/acts_as_customizable.rb

```diff
@@ -224,11 +224,7 @@ def reset_custom_values!
         end
 
         def custom_values_to_validate
-          if persisted?
-            @custom_values_to_validate ||= []
-          else
-            custom_field_values
-          end
+          @custom_values_to_validate ||= persisted? ? [] : custom_field_values
```

Comment:
This is a subtle change in logic. Now, a record that is validated multiple times will only consider custom values that were there for the first validation - since these are cached.

I don't know how often we have this case where a record is kept in memory for a while so that this is significant. But it could lead to hard-to-debug cases _if_ that happens.